common/web/lm-worker/src/main/correction/context-tracker.ts

```diff
@@ -532,17 +532,20 @@ export class ContextTracker extends CircularArray<TrackedContextState> {
     if(tokenizedContext.left.length > 0) {
       for(let i = this.count - 1; i >= 0; i--) {
         const priorMatchState = this.item(i);
-        if(transformDistribution && transformDistribution.length > 0) {
+        const priorTaggedContext = priorMatchState.taggedContext;
+        if(priorTaggedContext && transformDistribution && transformDistribution.length > 0) {
```

Comment:
We shouldn't need the null-guard now, but... just in case.  Don't want a mistake causing spurious keyboard errors in the mobile apps.